lib/tests/streamlit/runtime/caching/common_cache_test.py

```diff
@@ -273,12 +269,7 @@ def cached_widget():
 
             add_script_run_ctx(threading.current_thread(), orig_report_ctx)
 
-    @parameterized.expand(
-        [
-            ("memo", memo),
-            ("singleton", singleton),
-        ]
-    )
+    @parameterized.expand([("memo", memo), ("singleton", singleton)])
```

Comment:
A bunch of these `@parameterized` annotations were using way more newlines than necessary, so I collapsed them all.